docs/source/reference/core/function.rst

```diff
@@ -11,3 +11,13 @@ Function
    chainer.force_backprop_mode
    chainer.no_backprop_mode
    chainer.grad
+
+Wrapper Functions
+=================
+
+Wrapper functions are backward-able functions (probably differentiable), that take an array or a Chainer :class:`~chainer.Variable` as an argument and return a Chainer :class:`~chainer.Variable`, or a tuple of a Chainer :class:`~chainer.Variable`.
+Wrapper functions should not have learnable parameters when used in models and are usually not members of the :class:`~chainer.Chain`. Even if they do have learnable parameters, these are ignored by Chainer training, to prevent the functions from changing output due to adjustment of learned parameters during training.
```

Comment:
I don't think we have to mention the possibility of wrapper functions having learnable parameters, because Chainer does not assume such usage (correct me if I'm wrong).

How about:
"Wrapper functions do not have learnable parameters and are thus usually not members of Chain." (end of the paragraph)

Reply:
This wording was worked out with help from @toslunar . I think it should remain as is, to make it clear it can have learnable parameters, but they'll have no effect.